src/lib/commandBarConfigs/applicationCommandConfig.ts

```diff
@@ -79,5 +85,195 @@ export function createApplicationCommands({
     },
   }
 
-  return isDesktop() ? [textToCADCommand] : [textToCADCommand]
+  const addKCLFileToProject: Command = {
+    name: 'add-kcl-file-to-project',
+    displayName: 'Add file to project',
+    description: 'Add KCL file or 3D model to new or existing project.',
```

Comment:
```suggestion
    description: 'Add KCL sample or 3D model to new or existing project.',
```

Using the word "sample" here might be useful to users trying to search for that term, like our team.

Reply:
```suggestion
    description: 'Add KCL file, Zoo sample, or 3D model to new or existing project.',
```

Alternative wording in case you like that better.